packages/manager/freefax/app/cypress/integration/order.spec.js

```diff
@@ -0,0 +1,73 @@
+import Page from '../support/po';
+
+const { DashboardPage } = Page.FreefaxPage;
+
+describe('Order Freefax', () => {
+  const serviceId = '0033972250451';
+
+  beforeEach(() => {
+    cy.fixture('2api_freefax_details').as('fixture_2api_freefax_details');
+
+    cy.server();
+    cy.route(
+      'GET',
+      '/engine/apiv6/me',
+      'fixtures:me',
+    ).as('apiMe');
+
+    // cy.route('GET', `/engine/2api/freefax/${serviceId}/details`, 'fixtures:2api_freefax_details')
+    //   .as('2apiFreefaxDetails');
+
+    cy.get('@fixture_2api_freefax_details')
+      .then((json) => {
+        cy.route({
+          method: 'GET',
+          url: `/engine/2api/freefax/${serviceId}/details`,
+          status: 200,
+          response: json,
+          headers: {
+            'content-type': 'application/json',
+          },
+        })
+          .as('2apiFreefaxDetails');
+      });
+
+    cy.route(
+      'GET',
+      '/engine/apiv6/freefax.json',
+      'fixtures:apiv6_freefax',
+    );
+
+    cy.route(
+      'GET',
+      `/engine/2api/freefax/notifications/${serviceId}`,
+      'fixtures:2api_freefax_notifications',
+    );
+
+    cy.route({
+      method: 'GET',
+      url: `/engine/apiv6/freefax/${serviceId}/voicemail/routing`,
+      status: 200,
+      response: '"fax"',
+    })
+      .as('apiv6FreefaxVoicemailRouting');
+
+    cy.route('GET', `/engine/apiv6/freefax/${serviceId}/voicemail`, 'fixture:apiv6_freefax_voicemail')
+      .as('apiv6FreefaxVoicemail');
+  });
+
+  it('should display order', () => {
+    const dashboard = new DashboardPage(serviceId);
+    dashboard.visit();
+    dashboard.isVisible();
+
+    cy.wait([
+      '@apiMe',
+      '@2apiFreefaxDetails',
+      '@apiv6FreefaxVoicemailRouting',
+    ]);
+
+    // for test only
+    cy.get('#creditAmount').select('10');
```

Comment:
Why not use a `testId` like you intended to at https://github.com/ovh-ux/manager/pull/73/files#diff-02fde28b4652982e60a115a34781dd1fR157

Reply:
Removed, `order.spec.js` was for test purpose.